whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -47,16 +47,20 @@
 
     private final String trustedIpRanges;
 
+    private boolean dosFilterEnabled;
```

Comment:
Use final? Or will it be possible to enable later via JMX?

Reply:
Hmm yes it is but this is the value at startup.
Will change it to final.